### PR TITLE
Rename CSS Animation Worklet IDL file to match reffy-reports

### PR DESCRIPTION
**Pull request by foolip**
Renamed in https://github.com/tidoust/reffy-reports/commit/b82ba0b160f3e2ddf4a96ac838c52894021f6f85.

Closes https://github.com/web-platform-tests/wpt/pull/19287.
Closes https://github.com/web-platform-tests/wpt/pull/19288.

### CODE REVIEW

File: animation-worklet/idlharness.any.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
 // https://wicg.github.io/animation-worklet/
 
 idl_test(
-  ['animation-worklet'],
+  ['css-animation-worklet'],
```
Comment (stephenmcgruer) [inline animation-worklet/idlharness.any.js]: So the current url (and directory) is css-animationworklet, with no second '-'. I guess the question is whether AnimationWorklet is considered one word or two. Looks like it is two words in all the prose, so I would consider the Houdini directory (and the url) to be missing a hyphen. 

Cc @majido

**Review by stephenmcgruer — APPROVED** [review]

File: animation-worklet/idlharness.any.js (hunk 1 above)
Comment (foolip) [reply animation-worklet/idlharness.any.js]: Agree that does look a bit funny.

Comment (majido) [reply animation-worklet/idlharness.any.js]: Animation Worklet is considered two words. So "css-animation-worklet" is correct. The old `css-animationworklet` directory should really be renamed to `css-animation-worklet`. I will start a houdini discussion to see how we can do it with least breakage. 

So this change is fine. I will make sure we update WPT if we change the directory and URL.

Comment (majido) [reply animation-worklet/idlharness.any.js]: I have filed: https://github.com/w3c/css-houdini-drafts/issues/959

**Pull request merged**